lib/kast.js

```diff
@@ -246,6 +246,7 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
     const pt = {
       "115792089237316195423570985008687907853269984665640564039457584007913129639936": "pow256",
       "115792089237316195423570985008687907853269984665640564039457584007913129639935": "maxUInt256",
+      "115792089237316195423570985008687907853269984665640564039457584007913129639680": "maxUInt256 - 255",
```

Comment:
I'm not sure if this prettification isn't too specific?

Reply:
I think this one is not overly specific.

Reply:
We can leave this out if we think people will figure out quickly enough that this is `maxUInt256 - 255`. It took me a few minutes.

Is your concern scalability, i.e. that we'll have to add a whole bunch of `maxUInt256 - X` numbers? That's possible.

Reply:
Agree that this improves readability.

Reply:
> Is your concern scalability, i.e. that we'll have to add a whole bunch of maxUInt256 - X numbers? That's possible.

This is my concern yes. I'm happy to merge and then revisit if it starts becoming an issue.

> Could it be confused for a stack operation?

This is a valid point IMO.